lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -128,6 +136,27 @@ def self.download_file_from_tag(repository:, tag:, file_path:, download_folder:)
 
         download_path
       end
+
+      # Creates (or updates an existing) GitHub PR Comment
+      def self.comment_on_pr(project_slug:, pr_number:, body:, reuse_identifier:)
+        comments = github_client.issue_comments(project_slug, pr_number)
+
+        existing_comment = comments.detect do |comment|
+          comment.body.include?(reuse_identifier)
+        end
+
+        comment_body = prepare_comment_body(body, reuse_identifier)
+
+        if existing_comment.nil?
+          github_client.add_comment(project_slug, pr_number, comment_body)
+        else
+          github_client.update_comment(project_slug, existing_comment.id, comment_body)
+        end
+      end
```

Comment:
This method should return the `reuse_identifier` used by the comment, so that the calling action also returns it, so that the caller on the Fastfile can access it, especially if it was an automatically-generated UUID.

You should then document in the action that it returns that reuse_id used (there's a method provided by `Fastlane::Action` for documenting that, I think it's called `self.return_value`?)

Reply:
Addressed in 0db0d86a4cdf2e73695dba7224d44a5a58611fb2